examples/spec/query_spec.rb

```diff
@@ -0,0 +1,116 @@
+require_relative 'spec_helper'
+require_relative '../lib/query'
+
+RSpec.describe Query do
+  include RSpec::Hive::WithHiveConnection
+  include RSpec::Hive::QueryBuilderHelper
+
+  subject { described_class.new }
+
+  let(:schema) { subject.table_schema }
+
+  before { connection.execute(schema.create_table_statement) }
+
+  context 'if we have a partition' do
```

Comment:
@mknapik I add a fast example on partition, but this could probably be more clear.